en/ros/offboard_control.md

```diff
@@ -59,7 +59,7 @@ Small low power examples:
 Larger high power examples:
 * [Intel NUC](http://www.intel.com/content/www/us/en/nuc/overview.html)
 * [Gigabyte Brix](http://www.gigabyte.com/products/list.aspx?s=47&ck=104)
-* [Nvidia Jetson TK1](https://developer.nvidia.com/jetson-tk1)
+* [Nvidia Jetson TK1](https://developer.nvidia.com/jetson-tk1) 
```

Comment:
@bkueng This appears to be discontinued. Do you know of a natural successor? Or who else I could ask?

Reply:
I'm not completely up to date, but typically the TX2 is often used.